parser/index.ts

```diff
@@ -181,7 +195,7 @@ function parseTreeToTextAndRanges(tree: StackItem): [string, MarkdownRange[]] {
         const hasAlt = node.tag.match(/data-link-variant="([^"]*)"/)![1] === 'labeled';
         const rawLink = node.tag.match(/data-raw-href="([^"]*)"/);
         const linkString = rawLink ? unescapeText(rawLink[1]!) : src;
-
+        ranges.push({type: 'inline-image', start: text.length, length: 3 + (hasAlt ? 2 + unescapeText(alt?.[1] || '').length : 0) + linkString.length});
```

Comment:
Let's decrease complexity by splitting the logic into several lines.
```suggestion
        const start = text.length;
        const length = 3 + (hasAlt ? 2 + unescapeText(alt?.[1] || '').length : 0) + linkString.length;
        ranges.push({type: 'inline-image', start, length });
```